core/web/config_controller.go

```diff
@@ -25,14 +25,26 @@ func (cc *ConfigController) Show(c *gin.Context) {
 	jsonAPIResponse(c, cw, "config")
 }
 
+type ConfigV2Resource struct {
+	Config string `json:"config"`
+}
+
+func (c ConfigV2Resource) GetID() string {
+	return "config_v2_resource"
```

Comment:
Should this be a unique identifier? I see some implementations returning a field, and others generating UUIDs. Old config for example:
```go

// GetID generates a new ID for jsonapi serialization.
func (c ConfigPrinter) GetID() string {
	return utils.NewBytes32ID()
}
```

Reply:
I've noticed we use both too and the docs don't require UUIDs explicitly, only say the ID has to be unique per resource.

https://jsonapi.org/format/#document-resource-object-identification
https://github.com/manyminds/api2go#marshalidentifier

Reply:
Doesn't that mean that different configs should return different IDs? Like a hash or a UUID?

Reply:
It's not obvious to me and I can't find a definitive answer. I assumed resource is defined by a URI. But in any case, generating a UUID seems like a safer bet so let me change it.